.github/workflows/ignore-policy.rego

```diff
@@ -5,8 +5,9 @@ import data.lib.trivy
 default ignore = false
 
 ignore_cves := {
-  "CVE-2022-1471", # org.yaml:snakeyaml.1.33
-  "CVE-2016-1000027" # org.springframework:spring-web:5.3.29
+  "CVE-2016-1000027", # org.springframework:spring-web:5.3.29
+  "CVE-2022-40159", # commons-jxpath:commons-jxpath:1.3: used by spotless-2116170985 (strange name, you can find it if you execute `./gradlew dependencies`), i.e. only build-time dependency
+  "CVE-2022-40160", # commons-jxpath:commons-jxpath:1.3
```

Comment:
To be precise and keep track, I would prefer to add the information in both lines.

```suggestion
  "CVE-2022-40159", # commons-jxpath:commons-jxpath:1.3: used by spotless-2116170985 (strange name, you can find it if you execute `./gradlew dependencies`), i.e. only build-time dependency
  "CVE-2022-40160", # commons-jxpath:commons-jxpath:1.3: used by spotless-2116170985 (strange name, you can find it if you execute `./gradlew dependencies`), i.e. only build-time dependency
```